### ISSUE TITLE
[New Rule] Microsoft Exchange Server’s Unified Messaging Spawning Vulnerability - CVE-2021-26857 (2021 - March)

### ISSUE THREAD

**Issue by austinsonger** [elastic/detection-rules#972]
## Description

This rule detects CVE-2021-2685 and detects new processes being spawned by the Exchange UM service where that process has not previously been observed before.

### Target indexes

`filebeat-*

### Target Operating Systems

Windows

### Platforms

Microsoft Exchange Server

### Query
```
((process.parent.executable.text:(*\\UMWorkerProcess.exe) AND winlog.event_id:(\"4688\" OR \"1\")) AND (NOT (process.parent.executable.text:(*\\wermgr.exe OR *\\werfault.exe))))
```

### References
- https://www.volexity.com/blog/2021/03/02/active-exploitation-of-microsoft-exchange-zero-day-vulnerabilities/

## Example Data

```
{
   "author": [
      "Austin Songer"
   ],
   "actions": [],
   "created_at": "2021-03-04T04:19:02.092Z",
   "updated_at": "2021-03-04T04:19:02.092Z",
   "created_by": "asonger",
   "description": "This rule detects CVE-2021-2685 and detects new processes being spawned by the Exchange UM service where that process has not previously been observed before.",
   "enabled": true,
   "false_positives": [
      "Unknown At This Time"
   ],
   "filters": [],
   "from": "now-360s",
   "id": "bd858d70-7ca0-11eb-85a7-ad79931b44fb",
   "immutable": false,
   "index": [
      "filebeat-*",
      "logs-*",
      "winlogbeat-*"
   ],
   "interval": "5m",
   "rule_id": "a4697da4-df6f-4ff2-9dab-2d4ceb2a5678",
   "language": "lucene",
   "license": "",
   "output_index": ".siem-signals-default",
   "max_signals": 100,
   "risk_score": 47,
   "risk_score_mapping": [],
   "name": "Microsoft Exchange Server’s Unified Messaging Spawning Vulnerability - CVE-2021-26857",
   "query": "((process.parent.executable.text:(*\\UMWorkerProcess.exe) AND winlog.event_id:(\"4688\" OR \"1\")) AND (NOT (process.parent.executable.text:(*\\wermgr.exe OR *\\werfault.exe))))",
   "references": [
      "https://www.volexity.com/blog/2021/03/02/active-exploitation-of-microsoft-exchange-zero-day-vulnerabilities/"
   ],
   "meta": {
      "from": "1m",
      "kibana_siem_app_url": "https://c08d328f5cb84f12ac3dbf42a377beae.eastus2.azure.elastic-cloud.com:9243/app/security"
   },
   "severity": "medium",
   "severity_mapping": [],
   "updated_by": "asonger",
   "tags": [
      "Elastic",
      "Windows",
      "Initial Access",
      "T1190",
      "TA0001",
      "Exploit Public-Facing Application"
   ],
   "to": "now",
   "type": "query",
   "threat": [
      {
         "framework": "MITRE ATT&CK",
         "tactic": {
            "id": "TA0001",
            "reference": "https://attack.mitre.org/tactics/TA0001",
            "name": "Initial Access"
         },
         "technique": [
            {
               "id": "T1190",
               "reference": "https://attack.mitre.org/techniques/T1190",
               "name": "Exploit Public-Facing Application",
               "subtechnique": []
            }
         ]
      }
   ],
   "throttle": "no_actions",
   "version": 1,
   "exceptions_list": []
}
```

**Comment by peasead** [comment]
Hey, @austinsonger thanks for opening this rule (and #973).

Do you plan on making a PR? We're more than happy to, but didn't want to swoop in if you had something in the mix.

**Comment by brokensound77** [comment]
resolved by #979
